app/data/get-queries.js

```diff
@@ -20,7 +20,11 @@ const getQueries = async (sbi) => {
       payload: JSON.stringify({ query }),
       json: true
     })
-    return payload.data.customerQueryTicketsBySbi.customerQueryTickets
+
+    const queries = payload.data.customerQueryTicketsBySbi.customerQueryTickets
+    queries.sort((a, b) => new Date(b.timestamp) - new Date(a.timestamp))
```

Comment:
Added in this extra line, noticed that on the `view-all` queries page, the queries were arranged in ascending order of the timestamp, but I remember us discussing this and thinking that for the external users we'd want their most recent query at the top to make it easier to find the last query they opened.